internal/service/server/block_storage.go

```diff
@@ -668,6 +668,10 @@ func waitForBlockStorageDetachment(config *conn.ProviderConfig, id string) error
 			if err != nil {
 				return 0, "", err
 			}
+
+			if instance == nil {
+				return 0, "", fmt.Errorf("GetBlockStorage is nil")
```

Comment:
(trivial) It would be nice to set more messages for the error. 
something like an ex) "fail to get block storage instance, it doesn't exist" ?
(same for others)